src/verify/enum_event_category.sql

```diff
@@ -0,0 +1,10 @@
+-- Verify maevsi:enum_event_category on pg
+
+BEGIN;
+
+DO $$
```

Comment:
what does this doo?

Reply:
> These dollar signs ($$) are used for [dollar quoting](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-DOLLAR-QUOTING), which is in no way specific to function definitions. It can be used to replace single quotes enclosing string literals (constants) anywhere in SQL scripts.

https://stackoverflow.com/questions/12144284/what-are-used-for-in-pl-pgsql

Reply:
The dollar is used as a delimiter. So you don't have to escape every quote.